nearlib/test/integration.test.js

```diff
@@ -241,6 +241,245 @@ describe('with deployed contract', () => {
     });
 });
 
+describe('with promises', () => {
+    let contract, contract1, contract2;
+    let oldLog;
+    let logs;
+    let contractName = 'test_' + Date.now();
+    let contractName1 = 'test_' + Math.random();
```

Comment:
Should we use some seed?

Reply:
It should be random enough to avoid collisions.

Reply:
@evgenykuzyakov let's pull it into separate test module?

Reply:
Ok. I'll do this when I add more promise tests (for errors and joined promises)